cmd/convox/services.go

```diff
@@ -67,7 +67,8 @@ func init() {
 
 	stdcli.RegisterCommand(cli.Command{
 		Name:        "services",
-		Description: "manage services",
+		Aliases: []string{"resources"},
```

Comment:
Nice, didn't know you could do this! Can you implement it the other way, i.e. resources aliases "services"? That'd make `convox help` bit a bit cleaner

```
$ convox help | grep resources
  services, resources	manage resources [prev. services]
```